Doc/whatsnew/3.10.rst

```diff
@@ -219,7 +219,7 @@ have been incorporated. Some of the most notable ones:
           File "<stdin>", line 1
             {x,y for x,y in zip('abcd', '1234')}
              ^
-        SyntaxError: did you forget parentheses around the comprehension target?
+        SyntaxError: did you forget parenthesis around the comprehension target?
```

Comment:
Am inclined to confirm with @pablogsal before this change, if he didn't actually mean "parentheses" as in plural in this case. It seems to make sense to me as is but again, I dint author it.

Reply:
Thanks for checking @nanjekyejoannah !

Indeed, I meant plural. This is also an actual error message so even if we want to change it it should be changed in tended with the source :)

Reply:
```suggestion
        SyntaxError: did you forget parentheses around the comprehension target?
```